src/dbSta/src/dbReadVerilog.cc

```diff
@@ -179,7 +179,7 @@ class Verilog2db
 
   bool hasTerminals(Net* net) const;
   dbMaster* getMaster(Cell* cell);
-  dbModule* makeUniqueDbModule(const char* name);
+  dbModule* makeUniqueDbModule(Cell* cell, Instance* inst);
```

Comment:
Why change makeUniqueDbModule to take an instance as well as the cell ?
All makeUniqueDbModule does is advance the name until it does not clash with any other modules.
Not sure we need to pass in the Instance (or at least I don't see why).

Reply:
Yosys uniquifies module names using module_name.inst_name convention.   This is something that read_verilog can parse.   module_name-1 can't be read back with read_verilog.